tox.ini

```diff
@@ -7,6 +7,7 @@ passenv = LANG LC_CTYPE LC_ALL MSYSTEM
 deps =
     mock
     nose
+    toml
```

Comment:
This should no longer be needed due to the test extra

Reply:
I thought that as well, but it fails the build without it. I had thought that `mock` (line 8) should also be absent for the same reason, so I think there is something else at play here.

Reply:
Good point!  I'll investigate later.

Reply:
Turns out I wasn't actually using the `test` extra in the `tox.ini`!  (Probably because old versions of tox didn't have a convenient way of specifying which extras were needed.)